package.json

```diff
@@ -74,7 +74,8 @@
     },
     "cls-hooked": {
       "semver": "^5.7.2"
-    }
+    },
+    "form-data": "2.5.4"
```

Comment:
We might want to scope this to just `request` and see if we can bump `supertest` up to a newer versions (`supertest` is the other consumer of `form-data`).